src/components/pages/gallery/Collections.tsx

```diff
@@ -84,24 +91,30 @@ export default function Collections(props: CollectionProps) {
     }>({});
 
     const updateScrollObj = () => {
-        if (collectionRef.current) {
+        if (collectionWrapperRef.current) {
             const { scrollLeft, scrollWidth, clientWidth } =
-                collectionRef.current;
+                collectionWrapperRef.current;
             setScrollObj({ scrollLeft, scrollWidth, clientWidth });
         }
     };
 
     useEffect(() => {
         updateScrollObj();
-    }, [collectionRef.current]);
+    }, [collectionWrapperRef.current]);
 
     useEffect(() => {
-        if (!collectionRef?.current) {
+        if (!collectionWrapperRef?.current) {
             return;
         }
-        collectionRef.current.scrollLeft = 0;
+        collectionWrapperRef.current.scrollLeft = 0;
     }, [collections]);
 
+    useEffect(() => {
+        collectionChipsRef[selected]?.current.scrollIntoView({
+            inline: 'center',
+        });
+    }, [selected]);
+
```

Comment:
side effect, now whenever a collection is selected it moved to center of the screen? 

Do we want, or should I add logic to avoid this

Reply:
Can you add a screenshot here?